src/contracts/facilitators/gsm/Gsm.sol

```diff
@@ -451,13 +452,38 @@ contract Gsm is AccessControl, VersionedInitializable, EIP712, IGsm {
     _accruedFees += fee.toUint128();
     IERC20(UNDERLYING_ASSET).safeTransferFrom(originator, address(this), assetAmount);
 
-    IGhoToken(GHO_TOKEN).mint(address(this), grossAmount);
+    _handleGhoBought(grossAmount);
```

Comment:
i d just rename to the term we agree on. (e.g. `useGho(grossAmount)`)
same with the others